templates/base.html

```diff
@@ -52,6 +52,9 @@
             </li>
             <li class="nav-item">
               <a class="nav-link" href="/Mine">Mine</a>
+            </li>
+			<li class="nav-item">
```

Comment:
Align it with the rest pls
It looks better this way :)

Reply:
what's not aliigned?

Reply:
figured it out, tabs are a menace